pkg/generate/ack/runtime_test.go

```diff
@@ -163,8 +159,12 @@ func TestRuntimeDependency(t *testing.T) {
 
 	// ACK runtime 0.8.0 removed the unused UpdateCRStatus method from
 	// AWSResourceDescriptor
-	rd := new(acktypes.AWSResourceDescriptor)
-	rdType := reflect.TypeOf(rd)
+	rdType := reflect.TypeOf((*acktypes.AWSResourceDescriptor)(nil)).Elem()
```

Comment:
hmm, nice catch...

Reply:
`.Elem()` is supposed to emulate an instance of the pointer type. Not sure how it was working before, probably voodoo magic.